scripts/osx/testTemplate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+OF_DIR=`pwd`;
+echo $OF_DIR;
+TEST_DIR="$OF_DIR/scripts/osx"
+cd $TEST_DIR
```

Comment:
then we could even totally drop the script, and execute xcodebuild directly from the yml, no?